packages/firebase_app_check/firebase_app_check_web/lib/firebase_app_check_web.dart

```diff
@@ -28,11 +29,30 @@ class FirebaseAppCheckWeb extends FirebaseAppCheckPlatform {
 
   /// Called by PluginRegistry to register this plugin for Flutter Web
   static void registerWith(Registrar registrar) {
+    final instance =
+        FirebaseAppCheckPlatform.instance = FirebaseAppCheckWeb.instance;
```

Comment:
```suggestion
  final instance = FirebaseAppCheckPlatform.instance =
            FirebaseAppCheckWeb(app: Firebase.app(firebaseApp.name));
```

Reply:
We don't have access to the `firebaseApp` variable here

Reply:
here: https://github.com/firebase/flutterfire/pull/11625/files#diff-ced897c7c86bb9b2236d8676e1d80e16f3627802ef58545c1344676bdea434d8R38

Reply:
I think I did this correctly